components/TARGET_PSA/services/crypto/COMPONENT_SPE/psa_crypto_partition.c

```diff
@@ -1172,16 +1179,16 @@ static void psa_key_management_operation(void)
                 }
 
                 case PSA_CREATE_KEY: {
-                    psa_key_id_t id = 0;
-                    size_t max_bits = 0;
+                    psa_key_id_t id;
+                    id.owner = psa_identity(msg.handle);
 
-                    bytes_read = psa_read(msg.handle, 1, &id, msg.in_size[1]);
+                    bytes_read = psa_read(msg.handle, 1, &(id.key_id), msg.in_size[1]);
                     if (bytes_read != msg.in_size[1]) {
                         SPM_PANIC("SPM read length mismatch");
                     }
-                    bytes_read = psa_read(msg.handle, 2, &max_bits, msg.in_size[2]);
-                    if (bytes_read != msg.in_size[2]) {
-                        SPM_PANIC("SPM read length mismatch");
+
```

Comment:
previously there was:
bytes_read = psa_read(msg.handle, 2, &max_bits, msg.in_size[2]);
what was passed in msg.in_size[2]?

Reply:
`max_bits` is no longer part of the crypto API, so it was removed